core/src/components/datetime/datetime.tsx

```diff
@@ -1526,71 +1534,96 @@ export class Datetime implements ComponentInterface {
     ampmItems: PickerColumnItem[],
     use24Hour: boolean
   ) {
+    const isDayPeriodRTL = isLocaleDayPeriodRTL(this.locale);
+    return (
+      <ion-picker-internal>
+        {!use24Hour && isDayPeriodRTL && this.renderDayPeriodPickerColumn(ampmItems)}
+        {this.renderHourPickerColumn(hoursItems)}
+        {this.renderMinutePickerColumn(minutesItems)}
+        {!use24Hour && !isDayPeriodRTL && this.renderDayPeriodPickerColumn(ampmItems)}
```

Comment:
Any reason why you opted to do this instead of re-ordering with CSS? I believe we re-order with css in then month/year picker, though there may be some keyboard considerations with using `order` instead of physically changing the order of the DOM nodes.

Reply:
Only reason was because the picker columns are rendered in a popover, but an inline style works great here. Updated the approach!